.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "cpp/third_party/objecthash"]
+	path = cpp/third_party/objecthash
+	url = https://github.com/benlaurie/objecthash.git
```

Comment:
I know I proposed it, but I'm feeling guilty already, I'm not sure why... :stuck_out_tongue: